sklearn/pipeline.py

```diff
@@ -549,7 +561,8 @@ def make_pipeline(*steps):
     -------
     p : Pipeline
     """
-    return Pipeline(_name_estimators(steps))
+    memory = kwargs.pop('memory', None)
+    return Pipeline(_name_estimators(steps), memory=memory)
```

Comment:
Please ensue that an error is raised if there are other keyword args to make_pipeline. Easiest way to do that is just to forward **kwargs in entirety.